src/instances.ts

```diff
@@ -165,7 +166,15 @@ function successfulTypeScriptInstance(
       loader.context
     );
 
-    loader._module.errors.push(...errors);
+    /**
+     * Since webpack 5, the `errors` property is deprecated,
+     * so we can check if some methods for reporting errors exist.
+     */
+    if (!!module.addError) {
```

Comment:
Do we need the `!!` here?

```suggestion
    if (module.addError) {
```

Reply:
Fixed in https://github.com/TypeStrong/ts-loader/pull/1195/commits/f0a81318d255c18f11027e043d3bd56b13460bad